### DISCUSSION TITLE
Disabling GPU in 1.2.0 on Windows

### DISCUSSION THREAD

**Discussion by mritola** [chidiwilliams/buzz#1015] [Q&A]
Hi,

Buzz 1.2.0 is considerably slower for me than the previous version on Windows, and I suspect that the reason is the new GPU support and the fact that my ancient GPU isn't quite up to scratch. What's the easiest way to disable the use of the GPU and go back to how things were in the previous version? The obvious answer is, of course, to revert back to 1.1.0, but I'd still like to be able to use the turbo models.

Thanks,
Mika

**Comment by raivisdejus** [comment]
@mritola Latest development build adds `BUZZ_FORCE_CPU` environmental variable support that will disable the GPU.

See https://chidiwilliams.github.io/buzz/docs/preferences on available environment variables. Info on getting latest development build is here https://chidiwilliams.github.io/buzz/docs/faq#9-where-can-i-get-latest-development-version
